protocols/relay/src/behaviour/handler.rs

```diff
@@ -54,28 +61,28 @@ pub struct Config {
 
 pub enum In {
     AcceptReservationReq {
-        inbound_reservation_req: inbound_hop::ReservationReq,
```

Comment:
Is this necessary? I found the prefixing quite descriptive to quickly see, which protocol this is related to.